ranges/to-comparators.js

```diff
@@ -3,6 +3,6 @@ const Range = require('../classes/range')
 // Mostly just for testing and legacy API reasons
 const toComparators = (range, options) =>
   new Range(range, options).set
-    .map(comp => comp.map(c => c.value).join(' ').trim().split(' '))
+    .map(comp => comp.map(c => c.value.trim()))
```

Comment:
this isn't exactly the same thing - `[' a ', ' b ']` would have previously produced `['a ', ' b']` but will now produce `['a', 'b']`.

Reply:
all tests are passed